app/views/todos/_github_resource.html.erb

```diff
@@ -0,0 +1,26 @@
+<div>
+  <h3><%= raw t("queue_types.#{queue_type.to_s.singularize}").humanize.titleize.pluralize %> (<%= resources.size %>): </h3>
```

Comment:
We are using `queue_type.to_s.singularize` many times, maybe it's better to store it in a variable.